deploy/kubernetes/secret.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: apm-server-secret
+type: Opaque
+data:
+  cloud_auth: ""
+  elasticsearch_password: Y2hhbmdlbWUK
+  elasticsearch_username: YXBtLXNlcnZlci11c2VyCg==
```

Comment:
Where do these values come from?

Reply:
they're base64 encoded default values `apm-server-user/changeme` - they needed some non-empty value so that libbeat would allow apm-server to start.